gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 633
+cordaApiRevision = 634
```

Comment:
Since our branch will be in parallel to main, over the course they will increase this number independently anyway, so I think we could leave it now as it is. Once we do PR against main we can update this number if needed.